svg/painting/parsing/stroke-miterlimit-computed.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     xmlns:h="http://www.w3.org/1999/xhtml"
+     width="800px" height="800px">
+  <title>SVG Geometry Properties: getComputedValue().strokeMiterlimit</title>
+  <metadata>
+    <h:link rel="help" href="https://svgwg.org/svg2-draft/painting.html#StrokeMiterlimitProperty"/>
+    <h:meta name="assert" content="stroke-miterlimit computed value is as specified."/>
+  </metadata>
+  <g id="target"></g>
+  <h:script src="/resources/testharness.js"/>
+  <h:script src="/resources/testharnessreport.js"/>
+  <h:script src="/css/support/computed-testcommon.js"/>
+  <script><![CDATA[
+
+test_computed_value("stroke-miterlimit", "0");
+test_computed_value("stroke-miterlimit", "0.5");
+test_computed_value("stroke-miterlimit", "1");
+test_computed_value("stroke-miterlimit", "7.5");
```

Comment:
Can this particular testing framework test correct handling of invalid values? E.g., a negative value in this case should be rejected by the parser.

Reply:
Invalid values are tested in `*/parsing/*-invalid.svg`; these found browser bugs relating to [image-rendering](https://bugs.chromium.org/p/chromium/issues/detail?id=901669),  [stroke-opacity](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/19656838/), [stroke-dasharray](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/19656979/), [stroke-width](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/19656907/) and more.

[stroke-miterlimit-invalid.svg](https://wpt.fyi/results/svg/painting/parsing/stroke-miterlimit-invalid.svg?label=stable&aligned) tests that negative values and misformed numbers are rejected.